proof/refine/ARM_HYP/Arch_R.thy

```diff
@@ -1306,6 +1306,20 @@ lemma archThreadSet_VCPU_Some_corres[corres]:
   apply (simp add: arch_tcb_relation_def)
   done
 
+crunches dissociateVCPUTCB
+  for no_0_obj'[wp]: no_0_obj'
+  and ksCurThread[wp]: "\<lambda>s. P (ksCurThread s)"
+  (simp: crunch_simps wp: crunch_wps)
+
+lemma vcpuSwitch_corres'':
+  "vcpu' = vcpu
+   \<Longrightarrow> corres dc (\<lambda>s. (vcpu \<noteq> None \<longrightarrow> vcpu_at  (the vcpu) s) \<and> valid_arch_state s)
```

Comment:
Feel free to leave this as is, but is our official style to have the arrow in the previous line?
```suggestion
  "vcpu' = vcpu \<Longrightarrow>
   corres dc (\<lambda>s. (vcpu \<noteq> None \<longrightarrow> vcpu_at  (the vcpu) s) \<and> valid_arch_state s)
```

Reply:
Ah yeah I think what you suggest is the official style. I can change if people would like